src/Message/AbstractOrderRequest.php

```diff
@@ -2,22 +2,80 @@
 
 namespace MyOnlineStore\Omnipay\KlarnaCheckout\Message;
 
+use MyOnlineStore\Omnipay\KlarnaCheckout\Address;
+
 abstract class AbstractOrderRequest extends AbstractRequest
 {
     use ItemDataTrait;
+    use AddressTrait;
+
+    /**
+     * @param array $billingAddress
+     *
+     * @return $this
+     */
+    public function setBillingAddress($billingAddress)
+    {
+        $this->setParameter('billing_address', Address::fromArray($billingAddress));
+
+        return $this;
+    }
+
+    /**
+     * @param array $shippingAddress
+     *
+     * @return $this
+     */
+    public function setShippingAddress($shippingAddress)
+    {
+        $this->setParameter('shipping_address', Address::fromArray($shippingAddress));
+
+        return $this;
+    }
+
+    /**
+     * @return array
```

Comment:
Dit is altijd een `Address` ipv een `array`. De typehint in `getOrderData` kan dan ook vervallen